docs/reference/icons-emojis.md

```diff
@@ -6,8 +6,9 @@ icon: material/emoticon-happy-outline
 
 One of the best features of Material for MkDocs is the possibility to use [more
 than 10,000 icons][icon search] and thousands of emojis in your project 
-documentation with practically zero additional effort. Moreover, custom icons 
-can be added and used in `mkdocs.yml`, documents and templates.
+documentation with practically zero additional effort. Moreover, [custom icons 
+can be added](../../setup/changing-the-logo-and-icons/#additional-icons) and 
```

Comment:
Could you please move the link definition outside of the text? We define all links outside, so the text is better to read (as can be seen in line 13) 🚀

Reply:
Ok, done.